test/test_NumericRounder.py

```diff
@@ -16,10 +16,13 @@
     (Decimal('-1.12345'), Decimal('-1.123')),
     (None, None),
     ('str', 'str'),
+    (True, True),
+    (False, False)
```

Comment:
If you're amending anyway, could you add a trailing slash here to get a nice diff the next time we add test cases as well? 😇